### PR TITLE
Update messages.json

### PR DESCRIPTION
**Pull request by ghost**
update the messaging to refer to yinbi in beta

### CODE REVIEW

File: messages.json [hunk 1]
```diff
@@ -215,12 +215,12 @@
...
-        "content.desktop.body.free": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，购买蓝灯可免费获赠隐币！隐币每日发放量指数减少，即刻购买，一起分享当日隐币赠送。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。",
-        "content.desktop.body.main_screen.free": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，购买蓝灯可免费获赠隐币！隐币每日发放量指数减少，即刻购买，一起分享当日隐币赠送。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。",
-        "content.desktop.body.secondary_screen.free": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，购买蓝灯可免费获赠隐币！隐币每日发放量指数减少，即刻购买，一起分享当日隐币赠送。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。",
-        "content.desktop.body.pro": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，续费蓝灯可免费获赠隐币！即刻续费，一起分享当日隐币配送。\n\nYNB会根据当天发放隐币总量，当天购买/续费数量和购买/续费时长，在每日日终发放。",
-        "content.desktop.body.main_screen.pro": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，续费蓝灯可免费获赠隐币！即刻续费，一起分享当日隐币配送。\n\nYNB会根据当天发放隐币总量，当天购买/续费数量和购买/续费时长，在每日日终发放。",
-        "content.desktop.body.secondary_screen.pro": "蓝灯与隐币是合作伙伴，升级到蓝灯5.2后，续费蓝灯可免费获赠隐币！即刻续费，一起分享当日隐币配送。\n\nYNB会根据当天发放隐币总量，当天购买/续费数量和购买/续费时长，在每日日终发放。",
+        "content.desktop.body.free": "蓝灯与隐币是合作伙伴，隐币正在小范围公测，升级到蓝灯5.2后，购买蓝灯可免费获赠隐币！隐币每日发放量指数减少，即刻购买，一起分享当日隐币赠送。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。隐币正在小范围公测。公测结束后，将以更大规模发放。公测阶段购买蓝灯专业版开抢，分得更多隐币!",
```
Comment (hchood) [inline messages.json]: So we still want to include the 5.2 version here?

Comment (hchood) [reply messages.json]: Also I would suggest we bump up `display_frequency_seconds` to be between 2-3 days and a week.

**Comment by Ghostwithout** [comment]
Sorry this comment might be late and I know this is to urge purchase, but I am bit concerned with "公测" (public test): if this is public test, what does it make for previous two weeks auction? User might interpret that we say this just because we want to push purchase. 

Would you review an alternative below @ghostwith 
It targets to urge for early purchase, and push redeem ratio at yin.bi. 
For user already 5.2+:
蓝灯与隐币是合作伙伴，早期用户抢先购买/续费获得更多隐币；随着参与用户更多，每日到手配额将逐渐下降。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。隐币兑换码日终生成后，请到yin.bi兑换隐币！

For user less than 5.2, use:
蓝灯与隐币是合作伙伴，早期用户抢先购买/续费获得更多隐币；升级到蓝灯5.2，一起分享当日隐币赠送。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。隐币兑换码日终生成后，请到yin.bi兑换隐币！

**Comment by ghostwith** [comment]
@Ghostwithout looks good. Now that most people are 5.2+ we can just use the 

蓝灯与隐币是合作伙伴，早期用户抢先购买/续费获得更多隐币；随着参与用户更多，每日到手配额将逐渐下降。\n\nYNB会根据当天发放隐币总量，当天购买/续费蓝灯专业版的数量和和时长，您购买/续费蓝灯专业版的数量和时长在每日日终发放。隐币兑换码日终生成后，请到yin.bi兑换隐币！

**Comment by ghostwith** [comment]
@ghostwith @hchood Language updated

**Review by hchood — APPROVED** [review]

**Review by hchood — APPROVED** [review]

**Pull request merged**